applications/ALEapplication/python_scripts/python_solvers_wrapper_ale.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+
+def CreateSolver(main_model_part, custom_settings):
+
+    if (type(main_model_part) != KratosMultiphysics.ModelPart):
+        raise Exception("input is expected to be provided as a Kratos ModelPart object")
+
+    if (type(custom_settings) != KratosMultiphysics.Parameters):
+        raise Exception("input is expected to be provided as a Kratos Parameters object")
+
+    parallelism = custom_settings["problem_data"]["parallel_type"].GetString()
+    solver_type = custom_settings["solver_settings"]["solver_type"].GetString()
+
+    # Solvers for OpenMP parallelism
+    if (parallelism == "OpenMP"):
+
+        if (solver_type == "ALEMonolithic"):
+            solver_module_name = "ale_navier_stokes_solver_vmsmonolithic"
+
+        elif (solver_type == "ALEFractionalStep"):
+            solver_module_name = "ale_navier_stokes_solver_fractionalstep"
+
+        else:
+            raise Exception("the requested solver type is not in the python solvers wrapper")
```

Comment:
Here I suggest providing a bit more info, [see here for an example](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py#L35)